### DISCUSSION TITLE
Proposta de critérios para fundo em benefício a grupos subrepresentados

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#19] [Geral]
Olá,

Tendo em vista a aprovação, na [última assembleia da APyB (14 nov. 2021)](https://t.co/YZUm6X47Ah), da “criação de um fundo para apoio de grupos menos representados, tendo recursos oriundos de um percentual de sobras de recursos arrecadados de patrocínios na organização de eventos apoiados pela associação Python Brasil” e a necessidade de definição de regras, escopo e funcionamento deste fundo. @pyladies-brazil e @afropython apresenta à APyB e sua Diretoria Deliberativa a seguinte proposta, bem como convida à comunidade para debater e dar andamento na efetiva criação do fundo:

**0. Quais são os eventos apoiados pela associação Python Brasil?**
Todos os eventos cujo fluxo de caixa utilize a infraestrutura financeira da APyB. Essa regra incidirá a partir da Python Brasil 2021, e não haverá modificação nos montantes remanescentes de eventos anteriores.

**1. Quem vai executar/usar o dinheiro do fundo?**
Iniciativas da comunidade Python Brasil com objetivo de fornecer auxílio financeiro para viabilizar a participação de pessoas no evento Python Brasil e que tenham como público alvo grupos historicamente sub representados, a saber: pessoas que se identificam com o gênero feminino, negras, indígenas, com deficiência e LGBTQIA+, especialmente população trans, e pessoas economicamente vulneráveis.

Quem deverá decidir sobre o uso do dinheiro será um comitê de pessoas diversas com suporte e supervisão dos Conselhos Deliberativo e Fiscal e Diretoria da APyB.

**2. Que percentual dos recursos excedentes deveria compor o fundo?**
Destinar para o fundo o percentual mínimo de 50% dos recursos excedentes dos eventos.

**3. Avaliação de impacto das ações financiadas pelo fundo**
As organizações das edições da Python Brasil deverão coletar informações relativas ao perfil sociocultural das pessoas participantes e palestrantes do evento, preferencialmente seguindo o padrão usado nas edições de 2020 e 2021. Além disso, a abertura desses dados, no repositório [https://github.com/pythonbrasil/dados](https://github.com/pythonbrasil/dados), deverá ser atividade obrigatória nos 90 dias subsequentes ao encerramento do evento.

Assim, será possível a APyB realizar análise comparativa a fim de avaliar o impacto das iniciativas financiadas pelo fundo, consequentemente a manutenção do mesmo.

-----------------------------------

Autoria da proposta: @cecivieira @lipemorais

**Comment by rougeth** [comment]
Obrigado pela proposta @cecivieira @lipemorais! Algumas dúvidas/considerações:

>  Todos os eventos cujo fluxo de caixa utilize a infraestrutura financeira da APyB. Essa regra incidirá a partir da Python Brasil 2021, e não haverá modificação nos montantes remanescentes de eventos anteriores.

Acho que deveríamos limitar essa regra a apenas eventos em que a APyB cobra 10%, ou seja, eventos que cobram ingresso.

> Quem deverá decidir sobre o uso do dinheiro será um comitê de pessoas diversas com suporte e supervisão dos Conselhos Deliberativo e Fiscal e Diretoria da APyB.

O que será levado em consideração na decisão sobre o uso do dinheiro? Quem será responsável por formar o comitê e como?

> Assim, será possível a APyB realizar análise comparativa a fim de avaliar o impacto das iniciativas financiadas pelo fundo, consequentemente a manutenção do mesmo.

Podemos expandir esse tópico? Acho que devíamos definir na proposta como avaliaremos o impacto e quais decisões serão tomadas a partir dela.

**Comment by cecivieira** [comment]
@rougeth 
> O que será levado em consideração na decisão sobre o uso do dinheiro? Quem será responsável por formar o comitê e como?

Acredito que isso pode ser formado em assembleia extraordinária da APyB, ou considerar os grupos identitários atuais e convidar suas representantes para compor o comitê.

> Podemos expandir esse tópico? Acho que devíamos definir na proposta como avaliaremos o impacto e quais decisões serão tomadas a partir dela.

Podemos sim.

**Comment by cecivieira** [comment]
@gabubellon propusemos esses critérios para começar a conversar sobre o fundo.

**Comment by gabubellon** [comment]
@cecivieira @rougeth De acordo com o descrito.
Sobre esse item:
> O que será levado em consideração na decisão sobre o uso do dinheiro? Quem será responsável por formar o comitê e como?

Eu vejo 2 propostas:
1. Os representantes da PythonBR do ano elegem quem irá ficar responsável por esse dinheiro (além claro da própria APyB)
2. Podemos ter um "Conselho" eleito em assembleia com uma mandato para ser o responsável por isso.

Acho que podemos sim @cecivieira  discutir em assembleia isso.
Acho de devemo chamar essa assembleia o quanto anos, imagino que até o final de Abril. Para isso eu faria um informe a lista e no informe oficial da APyB, e já tendo a agenda desse assembleia, para que a mesma foque apenas nos assuntos pertinentes e o assunto não se desvie.

**Reply by rougeth** [reply]
Ainda que seja escolhido um conselho/comitê, com base em que esses grupos devem tomar a decisão?

Eu acho que ao invés da APyB transferir a responsabilidade da escolha, eu acho que seria mais interessante a APyB seguir uma série de critérios para aprovar ou recusar uma proposta que venha da comunidade.

Um desses critérios poderia ser que a proposta precisa ser discutida abertamente (pelo menos em algum momento). Dessa forma outras comunidades (além das PyLadies e AfroPython, por exemplo) tenham oportunidade de participar.

**Reply by gabubellon** [reply]
Mas quando eu digo `conselho/comitê`, estou dizendo uma estrutura formada dentro da APyB, hoje se os Conselhos ou Diretoria tomam decisões, a decisão é da APyB.

> Podemos ter um "Conselho" eleito em assembleia com uma mandato para ser o responsável por isso.

Esse item, podemos ter dentro do mandato da diretoria e conselheiros, um ¨conselho de fundo de apoio", formalmente formdo pela direção financeira, presidentes dos conselhos e +X entidades externas (podendo ser pessoas eleitas pela organização da Python Brasil do qual o fundo existe).

Dessa forma esse conselho estará sempre dentro da APyB (podendo ter a rotação dos mesmos, uma vez que as chapas mudam a cada 2 anos)e ter pessoas que deram origem aquele dinheiro. E ainda assim estará sobre todas as regras da APyB, uma vez que Conselhos Fiscasi e Deliberativos tem que excerser suas funções sobre as decisões.

**Reply by rougeth** [reply]
Eu acho que poderia ser mais simples que isso. Se a proposta vier da comunidade e a APyB tiver uma lista de critérios para avaliar, não precisaríamos de um comitê/estrutura/eleição/etc para tomar a decisão.

**Comment by rougeth** [comment]
Abrindo uma outra thread sobre esse assunto. A aplicação do fundo esta condicionada a situação financeira da APyB. Dito isso, é importante a APyB definir as regras dessa condição e conseguir responder se o caixa da APyB está saudável ou não.

Por exemplo, ter o caixa no positivo, na minha opinião, não é o suficiente para dizer que a situação é saudável. Seria ter caixa suficiente para bancar uma Python Brasil? (Isso foi o objetivo durante algumas gestões da APyB).

cc @apyb/diretoria

**Reply by gabubellon** [reply]
@rougeth Gostaria aqui de ter uma opinião da contabilidade na visão de "saudável" deles. 
Eu acho que `caixa suficiente para bancar uma Python Brasil` não é mais apenas a única regra.

**Comment by cecivieira** [comment]
Bom dia!
Gostaria de retomar essa conversa para verificar a viabilidade de uso desse recurso para bolsa de viagem para participante e palestrante da Python Brasil 2022.

Alguma sugestão de como podemos encaminhar @apyb @gabubellon @alynnefs (coloquei as conselheiras que lembrei)?

**Reply by gabubellon** [reply]
@apyb/conselho-deliberativo @apyb/diretoria Ao meu ver precisamos ter uma reunião diretoria para ter ok sobre isso e posteriomermente uma assembleia para aprovação dos membros, mas queria a visão de vocês

**Reply by rougeth** [reply]
Acho que meus pontos ainda estão abertos:

1. Critérios para quais iniciativas receberão os apoios
2. Valores/porcentagens variáveis de acordo com a situação financeira da APyB
3. Como e quando vamos avaliar o impacto dessa iniciativa

Dito isso, como o uso do dinheiro que sobrou especificamente do ano passo já foi aprovado em assembleia, talvez não precisemos definir tudo isso antes para usar nas bolsas.

**Reply by gabubellon** [reply]
@rougeth mas você está esperando uma resposta desses pontos por quem ? Pela organização de 2021 ? Pela diretoria da Apyb ? Por ambos ? Você tem alguma proposta sobre esses pontos ?
Eu mesmo pontuei sobre eles aqui https://github.com/apyb/comunidade/discussions/19#discussioncomment-2504022 
Agora se esses pontos precisam ser debatidos e fechados acho que o correto então é o mesmo ser feito em um encontro que a pauta seja essa.
Para mim todos os 3 itens tem que vir da APyB (óbvio com apoio de quem está propondo) pois principalmente os itens 1  e 2 serão base para uso futuro
O que proponho é :
* Uma reunião de diretoria e conselho e prepotentes disso com foco em propor e aprovar os pontos faltantes
* Uma comunicação ao membros da APyB sobre o que foi acordado.

cc @apyb/conselho-deliberativo  @apyb/diretoria

**Reply by gabubellon** [reply]
Sobre o uso do dinheiro do ano passado, também não vejo que precisamos esperar definir tudo isso para começar o uso. 
@cecivieira sabe se a PyLadies e/ou Afro Python já tem algum plano de uso desenhado ? Acho que esse seria um bom momento para te-lo e começar o mesmo.

**Comment by rougeth** [comment]
> mas você está esperando uma resposta desses pontos por quem?

De quem está propondo o fundo.

> Você tem alguma proposta sobre esses pontos?

Não tinha, mas depois que você perguntou, surgiu uma ideia:

1. Fundo com um único objetivo: levar grupos sub-representados a Python Brasil 
2. Dinheiro composto por 50% do valor que sobrar dos eventos que cobram por ingresso
3. Aprovação anual do fundo condicionada a situação financeira da APyB (definição a ser feita pela direção da APyB)
4. Definimos que PyLadies e AfroPython serão as comunidades que terão acesso ao fundo (50% cada). Com isso, simplificamos o processo e evitamos a necessidade de conselhos, eleição, mandato, etc. Outras comunidades podem ser adicionadas posteriormente.
5. As comunidades com acesso ao fundo definirão como e com quem o dinheiro será usado, seguindo a única regra descrição do item 1. Ou seja, o item `1. Quem vai executar/usar o dinheiro do fundo?` deixa de fazer parte da proposta, nesse caso, a APyB delega essa definição as comunidades escolhidas.
6. No final de cada ano, as comunidades farão um relatório descrevendo como o fundo foi usado (como fazemos com o apoio da PSF). Liberação do fundo condicionada a escrita do relatório do ano anterior.

Na minha opinião, dessa forma, o processo não fica burocrático e não aumentamos a carga de trabalho para direção da APyB. Repeti os dois primeiros items que já estavam definidos no texto inicial. 

@cecivieira o que acha?

**Reply by cecivieira** [reply]
Sugestões de acréscimo na proposta:

1 - O recebimento do dinheiro está condicionado aos requisitos fiscais para saída de dinheiro da Associação.

2 - A comunidade a ser beneficiada pode recusar o recebimento do dinheiro.

3 - Caso uma das comunidades beneficiada não se manifeste em favor do recebimento, após 365 dias contados a partir da comunicação de doação da APyB , o montante será endereçado a outra comunidade. No caso de nenhuma das comunidades se manifeste a favor, o dinheiro será acumulado.

**Reply by rougeth** [reply]
> 3 - Caso uma das comunidades beneficiada não se manifeste em favor do recebimento, após 365 dias contados a partir da comunicação de doação da APyB , o montante será endereçado a outra comunidade.

Como o objetivo é levar pessoas para Python Brasil, talvez seja melhor que o limite considere a data do evento, algo como "3 meses antes da Python Brasil do ano".

> No caso de nenhuma das comunidades se manifeste a favor, o dinheiro será acumulado.

Caso não seja usado, na minha opinião, o valor deveria voltar para o caixa da APyB. Isso simplifica o lado da associação e cria a necessidade das comunidades engajarem na iniciativa, já que caso o dinheiro não seja usado, o fundo perde o propósito.

**Reply by cecivieira** [reply]
> Como o objetivo é levar pessoas para Python Brasil, talvez seja melhor que o limite considere a data do evento, algo como "3 meses antes da Python Brasil do ano".

Poderia reduzir para 4 meses o prazo para manifestação após 1. contato da APyB com a comunidade. Assim, simplifica a aplicação da regra, pois conta o tempo a partir o início do processo e não do fim dele (dado que o uso do dinheiro 3 meses antes do evento já deveria estar sendo executado).

>Caso não seja usado, na minha opinião, o valor deveria voltar para o caixa da APyB. Isso simplifica o lado da associação e cria a necessidade das comunidades engajarem na iniciativa, já que caso o dinheiro não seja usado, o fundo perde o propósito.

Também pensei nisso, pelos mesmo motivos que os teus. No entanto, pensando que as comunidades oscilam um pouco o nível de atividade, os períodos de baixa impactarão diretamente na disponibilização de dinheiro nos períodos de alta, ou seja, quando a comunidade estiver mobilizada não terá o suporte financeiro do fundo, devido a inatividade do período anterior. 

Concordo que o mais fácil é devolver o dinheiro pro caixa da APyB. Talvez a gente precise pensar um pouco melhor sobre esse ponto.

**Reply by rougeth** [reply]
> Concordo que o mais fácil é devolver o dinheiro pro caixa da APyB. Talvez a gente precise pensar um pouco melhor sobre esse ponto.

Hoje não temos um processo definido para manter um caixa separado, precisaríamos ver isso antes. Se vocês não enxergam esse ponto como um requisito mínimo para o fundo, minha sugestão é revisitar no futuro e seguirmos no que for mais simples agora.

**Comment by cecivieira** [comment]
Como consolidação desse topico de discussão, informo a versão final dos critérios para uso do fundo. Solicito aprovação por parte da @apyb/diretoria e divulgação da decisão tomada.

-----------------

# Fundo para Inclusão

O **Fundo para Inclusão** é um fundo mantido pela Associação Python Brasil (APyB) para apoiar a participação de grupos historicamente sub representados nas edições presenciais do evento "Python Brasil".

O fundo é composto por rescursos excedentes dos eventos organizados e/ou apoiados pela APyB e será executado pelas comunidades AfroPython e PyLadies.

## 1. OBJETIVO

O fundo tem como único objetivo proporcionar a participação presencial de integrantes de grupos historicamente sub representados no evento "Python Brasil".

**Parágrafo único:** Entendemos que a composição dos **Grupos historicamente sub representados** se dá por mulheres, pessoas negras, indígenas, com deficiência, comunidade LGBTQIA+, especialmente população trans, e pessoas economicamente vulneráveis.

## 2. DA COMPOSIÇÃO

O fundo é composto por 50% de todo o valor excedente do caixa dos eventos, a partir da sua conclusão.

**Parágrafo único:** Os eventos elegíveis para composição do fundo são os que utilizam a estrutura financeira da APyB e que estejam sujeitos a cobrança da taxa de 10% por parte da Associação.

## 3. DA DISTRIBUIÇÃO

Os recursos do fundo serão distribuídos igualmente entre as comunidades participantes.

3.1. No primeiro trimestre de cada ano, junto com sua prestação de conta anual, a APyB publicará a totalidade dos recursos disponíveis no fundo, bem como o montante destinado a cada comunidade.

3.2. As comunidades beneficiadas pelo fundo deverão manifestar interesse no seu uso no prazo de 4 meses a contar da publicação da prestação de contas anual.

3.2.1. Caso apenas uma das comunidades manifeste interesse, todo o montante será destinado a ela.

3.2.2. Caso nenhuma das comunidades manifeste interesse no uso dos recursos, o montante será direcionado para o caixa geral da APyB.

## 4. DO USO

Os recursos do fundo poderão ser usados até a conclusão da edição da Python Brasil vigente.

4.1. O uso dos recursos está condicionado ao cumprimento dos requisitos fiscais da APyB para execução financeira.

4.2. As representantes das comunidades beneficiadas junto à APyB devem ser membras associadas da APyB na data em que o fundo for destinado.

## 5. DA PRESTAÇÃO DE CONTAS

Anualmente as comunidades beneficiadas pelo Fundo deverão prestar contas à APyB através do preenchimento de formulário fornecido pela associação, a fim de coletar dados do uso dos recursos e impacto da iniciativa no evento "Python Brasil". 

**Parágrafo único:** O preenchimento do formulário é obrigatório e será condição para que a comunidade possa participar do fundo no próximo ano.

**Comment by cecivieira** [comment]
Esses critérios foram aprovados? @apyb/diretoria

**Reply by cecivieira** [reply]
Precisamos dessa resposta para decidir se lançar ou não o Programa de Auxílio Financeiro.

**Reply by gabubellon** [reply]
Imagino que isso tenha que ir para uma reunião da mesma se ser aprovada o mais aprovada o mais rápido possível (se assim for possível) pra implementação disso ainda esse ano

**Reply by alynnefs** [reply]
@gabubellon reunião geral com os associados ou só diretoria? Pq pelo cronograma sugerido vai ficar bem corrido

**Reply by gabubellon** [reply]
Diretoria apenas, não vejo necessidade de associados uma vez que isso já foi aprovado em assembleia

**Reply by tiidadavena** [reply]
A próxima reunião da diretoria será próxima segunda-feira (22/08), vou verificar a possibilidade de uma reunião extraordinária com a @apyb/diretoria

**Comment by cecivieira** [comment]
Juliana,
Esse tema não poderia entrar na pauta da reunião de segunda-feira (22)?

**Reply by tiidadavena** [reply]
Oi, Ceci, esse tema já foi incluso na [pauta da próxima reunião](https://github.com/apyb/tarefas/issues/149), a pergunta era mais para saber se vocês conseguem esperar até lá ou se haveria a necessidade de uma reunião extraordinária

**Comment by cecivieira** [comment]
Ju, podemos esperar até a reunião de segunda (22).

**Comment by muriloviana** [comment]
> 4.2. As representantes das comunidades beneficiadas junto à APyB devem ser membras associadas da APyB na data em que o fundo for destinado.

Nesta parte acho que seria interessante reforçar, para evitar dúvidas, que as comunidades permitidas de movimentar esse fundo, no momento, são AfroPython e PyLadies. O que acham?

De resto, ficou muito bom! Achei de fácil e claro entendimento. Parabéns as pessoas envolvidas 👏

**Comment by cecivieira** [comment]
Concordo, Murilo.

**Comment by cecivieira** [comment]
Boa noite!!!

Temos resposta sobre a aprovação do fundo?

Lançamos o Programa de Auxílio Financeiro e no dia 04set. está prevista a seleção das pessoas a serem beneficiadas. A quantidade de pessoas beneficiadas está diretamente atrelada a esta discussão.

**Reply by rougeth** [reply]
Sim! 🎉 O texto foi aprovado sem nenhuma alteração, vamos publicá-lo no site e preparar os anúncios, mas vocês podem seguir com os trabalhos.

Obrigado @cecivieira, @lipemorais e quem mais contribuiu de alguma forma para a criação do fundo!

**Comment by cecivieira** [comment]
@apyb/diretoria Como será feita a publicação do montante atual do fundo? Nos critérios prevê a publicação junto com a prestação de conta do ano anterior, mas como será dada essa primeira publicação ?

@muriloviana @tiidadavena @rougeth

**Comment by tiidadavena** [comment]
@cecivieira foi levantada a pergunta se pessoas físicas podem contribuir com fundo

**Reply by tiidadavena** [reply]
No caso, contribuir com crowdfunding das PyLadies BR com a Afropython? Porque vi que empresas podem contribuir com o Apydrinhamento, mas como PF podem contribuir para ajudar?

**Reply by cecivieira** [reply]
@tiidadavena o [Fundo para Inclusão](https://python.org.br/fundo-para-inclus%C3%A3o/) tem como fonte financeira apenas o dinheiro excedente dos eventos que usam a estrutura da APyB e que se encaixam nos critérios estabelecidos nas regras do Fundo.

O [Programa de Auxílio Financeiro para a Python Brasil 2022](https://github.com/pythonbrasil/pybr2022-bolsa-viagem/) tem outras fontes financeiras. As empresas podem contribuir através desse [Plano de Apoio](https://www.canva.com/design/DAFJmcSHEL8/xIJW94m3geO31p2Q_eHLhA/edit?utm_content=DAFJmcSHEL8&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton), já pessoa física, preciso saber de @muriloviana como ela poderia contribuir.

Também tem outra pessoa interessada em fazer esse tipo de contribuição.

**Reply by muriloviana** [reply]
@cecivieira confesso a você que não sei como podemos proceder, pois como se trata de uma doação de PF precisaríamos de alguma forma comprovar essa doação feita pela pessoa, pois isso inclusive entra na declaração anual do IRPF.

Verificarei com a contabilidade da APyB certinho como podemos viabilizar esse tipo de doação. Lembrando que sugestões são sempre bem-vindas 😉

**Reply by gabubellon** [reply]
Vou dar meu pitaco:
* Fundo de Inclusão é uma coisa e está bem definido que é o excedente dos eventos, aqui eu acho que ninguém nem PF nem PJ pode ajuda
* Programa de Auxilio Financeiro ao evento eu vejo que é o local certo e ai velho alguns pontos:


Os pontos:
- Consolidar as pessaos que não forem incluidas no Fundo de Inclusão;
- Para essas pessoas pode-se entrar em contato com as PFs que querem contribuir  diretamente para essa pessoa, ou para o grupo (podendo utilizar alguma plataforma de apoio estilo apoia-se)
- Pode-se pedir o pagamento de um item, como a passagem , a hospedagem, isso dai eu faria de maneira direta , tendo uma lista de PFs que querem ajudar e enviando diretamente a elas o valor/item que precisa ser pago pra alguém
- Eu acho que pra doação PF->PF com valor baixo (até uns 2mil) não precisamos envolvr a APyB na transação, ficando a cargo que quem está organizando essas doação receber e organizar as prestações de conta (podendo até manter a PF anonima se ela assim quiser)
- PJ que quiser doar, doa pra Org/Apyb com NF certinho ai a Org manda esse dinheiro pro Programa de Auxilio

**Comment by cecivieira** [comment]
@gabubellon acredito que não é uma boa prática dar dados pessoas para ninguém. A pessoa fez inscrição em um programa com regras explícitas. Acredito que não é interessante fazer esse direcionamento.

Também vou pesquisar como poderia ser recebida essa doação de uma PF para um PJ, mas acredito que a contabilidade é a melhor área para responder.

**Reply by gabubellon** [reply]
@cecivieira eu não penso em dar dados pessoais. mas fazer a ponte. Com algo controlado e documentado a todos, tipo;
- Uma pessoa precisa de R$XPTO para comprar tal passagem, se puder contibrui envie o$$ pra <contato centralizando tudo> e dai se devolve a pessoa  a prestação de contas.
Mas eu acho que essa é uma ultima opção dentre algo formal contábil ou até mesmo um site vakinha antes

**Comment by cecivieira** [comment]
Massa
Entendi. Vamos consolidar a proposta com o que foi conversado aqui e em
breve posto a nova versão.
